app/views/devise/shared/_links.html.erb

```diff
@@ -18,8 +18,8 @@
   <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name) %><br />
 <% end %>
 
-<%- if devise_mapping.omniauthable? %>
-  <%- resource_class.omniauth_providers.each do |provider| %>
-    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider) %><br />
-  <% end %>
+#<%- if devise_mapping.omniauthable? %>
```

Comment:
To jest źle wykomentowane. tak naprawde nie ma komenta tylko wstawiony jest #

Reply:
unnecessary element, deleted